### ISSUE TITLE
Fix simple html issues from the w3 Validator

### ISSUE THREAD

**Issue by lukasjuhrich** [agdsn/sipa#405]
When the [w3 Validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fagdsn.de%2Fsipa%2Fnews%2F) against `https://agdsn.de`, it catches some technical and/or accessibility errors which are simple to fix.  Multiple endpoints (especially those with forms) should be checked.
